cves/CVE-2015-5964.yml

```diff
@@ -303,10 +305,10 @@ CWE_instructions: |
 
   Just the number here is fine. No need for name or CWE prefix. If more than one
   apply here, then choose the best one and mention the others in CWE_note.
-CWE:
+CWE: 399
```

Comment:
Should this be a specific CWE, or is just a CWE category ok? Because this is just a category that a number of other CWEs fall into.

Reply:
This was the cwe listed at https://nvd.nist.gov/vuln/detail/CVE-2015-5964